### PR TITLE
Small editorial changes

### PR DESCRIPTION
**Pull request by paulehoffman**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -206,8 +207,8 @@ developed a large amount of expertise in using AWS effectively, and retains an
 equity stake in A-Cloud.
 
 However, one can imagine applications which don't need much more than a virtual 
-Linux instance, in cases where Azure has a region in closer geographic proximity 
-to a large retail site than any A-Cloud region; in this case Azure might be a great
+Linux instance, in cases where Azure or Google Could has a region in closer geographic proximity
```
Comment (timbray) [inline README.md]: Typo on GCloud's name

File: README.md [hunk 2]
```diff
@@ -178,7 +179,7 @@ The vast majority of IT people have come to understand the advantages of moving
 app deployments from their own infrastructure to the public cloud, including 
 increased agility, world-class security, and reduced costs. Now, when they approach
 leadership with the idea of moving to the cloud, they will no longer have to deal with 
-fear that they might be empowering a competitor.
+fear that they might be perceived as empowering a competitor.
```
Comment (timbray) [inline README.md]: Nope. It's not just perception, it's hard cold cash.  But I like all the other edits!

**Review by timbray — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (paulehoffman) [reply README.md]: Agree

File: README.md (hunk 2 above)
Comment (paulehoffman) [reply README.md]: Fair point.

**Pull request merged**